src/globals.c

```diff
@@ -607,8 +608,8 @@ void load_flags_in_config(char *command, struct option *opts_array, const struct
 		config_found = config_parse(config_file, config_loader_set_opt, &loader_data);
 		free_string(&config_file);
 		if (config_found) {
-			debug("Using configuration file %s\n", tok);
-			break;
+			debug("Loading from configuration file %s\n", tok);
+			continue;
```

Comment:
you don't need a continue here, it's the last command of the for

Reply:
btw, This line isn't being printed for me when I run

Swupd diagnose --debug

Reply:
> you don't need a continue here, it's the last command of the for

true ☺️

Reply:
> btw, This line isn't being printed for me when I run
> 
> Swupd diagnose --debug

oh you are right, that is not going to show because regardless of if the user has chosen --debug or not, we haven't parsed that value yet. So I guess there is no point on adding those messages if they are never going to be printed. I'll remove them.

Reply:
that's a chicken egg problem